ui/app/templates/vault/cluster/init.hbs

```diff
@@ -1,20 +1,32 @@
 <SplashPage as |Page|>
   {{#if keyData}}
     <Page.header>
-      <h1 class="title is-4">
-        Vault has been initialized! {{#if (eq keyData.keys.length 1)}}
-        Here is your key.
-        {{else}}
-        Here are your {{pluralize keyData.keys.length "key"}}.
-        {{/if}}
-      </h1>
+      {{#with (or keyData.recovery_keys keyData.keys) as |keyArray|}}
```

Comment:
I believe `with` is deprecated in favor of [`let`](https://emberjs.com/api/ember/3.4/classes/Ember.Templates.helpers/methods/let?anchor=let) now.

Reply:
Ooo and it can yield multiple items 😙👌